assets/css/layout/header.css

```
@@ -54,61 +58,79 @@
+          transform: skewX(-50deg);
         }
       }
 
       li {
-        @apply lg:mr-4 lg:text-center lg:bg-drk-green-octo text-white;
+        @apply lg:mr-4 lg:text-center text-white;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        text-align: center;
         font-size: rfs(16px);
```

Biome

[error] Unexpected unknown function: rfs

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)